projects/packages/forms/src/modules/form/view.js

```diff
@@ -0,0 +1,215 @@
+		},
+
+		get showFromErrors() {
+			const context = getContext();
+
+			return ! state.isFormValid && context.showErrors;
+		},
+
+		get getFormErrorMessage() {
+			const config = getConfig( NAMESPACE );
+			if ( state.isEmptyForm ) {
+				return config.error_types.invalid_form_empty;
+			}
+			return config.error_types.invalid_form;
+		},
+
+		get getErrorList() {
+			const errors = [];
+			if ( state.isEmptyForm ) {
+				return errors;
+			}
+			const context = getContext();
+			if ( context.showErrors ) {
+				Object.values( context.fields ).forEach( field => {
+					if ( field.error && field.error !== 'yes' ) {
+						errors.push( {
+							anchor: '#' + field.id,
+							label: field.label + ' : ' + getError( field ),
+							id: field.id,
+						} );
+					}
+				} );
+			}
+			return errors;
+		},
+
+		get getFieldValue() {
+			const context = getContext();
+			const fieldId = context.fieldId;
+			const field = context.fields[ fieldId ];
+			return field.value;
+		},
+	},
+
+	actions: {
+		updateFieldValue: ( fieldId, value ) => {
+			updateField( fieldId, value );
+		},
+		handleChangeField: withSyncEvent( event => {
+			let value = event.target.value;
+			const context = getContext();
+			const fieldId = context.fieldId;
+
+			if ( context.fieldType === 'checkbox' ) {
+				value = event.target.checked ? '1' : '';
+			}
+
+			updateField( fieldId, value );
+		} ),
+
+		handleOnInputField: withSyncEvent( event => {
+			const value = event.target.value;
+			const context = getContext();
+			const fieldId = context.fieldId;
+			const field = context.fields[ fieldId ];
+
+			if ( field.type === 'checkbox' ) {
+				field.value = event.target.checked ? value : '';
+			} else {
+				field.value = value;
+			}
+
+			field.error = validateField( field.type, field.value, field.isRequired, field.extra );
+		} ),
+
+		handleMultipleChangeField: withSyncEvent( event => {
+			const context = getContext();
+			const fieldId = context.fieldId;
+			const field = context.fields[ fieldId ];
+			const value = event.target.value;
+			let newValues = [ ...( field.value || [] ) ];
+
+			if ( event.target.checked ) {
+				newValues.push( value );
+			} else {
+				newValues = newValues.filter( v => v !== value );
+			}
+
+			updateField( fieldId, newValues );
+		} ),
+
+		handleBlurField: withSyncEvent( event => {
+			const context = getContext();
+			updateField( context.fieldId, event.target.value, true );
+		} ),
+
+		formSubmit: withSyncEvent( event => {
+			const context = getContext();
+
+			if ( ! state.isFormValid ) {
+				context.showErrors = true;
+				event.preventDefault();
+				event.stopPropagation();
+			} else {
+				context.isSubmitting = true;
+			}
+		} ),
+	},
+
+	callbacks: {
+		initializeField() {
+			const context = getContext();
+			const { fieldId, fieldType, fieldLabel, fieldValue, fieldIsRequired, fieldExtra } = context;
+			registerField( fieldId, fieldType, fieldLabel, fieldValue, fieldIsRequired, fieldExtra );
+		},
```

Comment:
There are ways to populate the parent context in PHP without having to do it on page load. That can be especially useful if you need to access values of that array to populate the HTML during server-side rendering.

If you want to investigate them at some point, let me know and we'll take a look.